test-suite/src/update.rs

```diff
@@ -109,17 +109,17 @@ test_case!(update, {
     g.run("INSERT INTO ErrTestTable (id) VALUES (1),(9);").await;
 
     let error_cases = [
-        (
-            "UPDATE TableA INNER JOIN ErrTestTable ON 1 = 1 SET 1 = 1",
-            Err(TranslateError::JoinOnUpdateNotSupported.into()),
-        ),
-        (
-            "UPDATE (SELECT * FROM ErrTestTable) SET 1 = 1",
-            Err(
-                TranslateError::UnsupportedTableFactor("(SELECT * FROM ErrTestTable)".to_owned())
-                    .into(),
-            ),
-        ),
+        // (
+        //     "UPDATE TableA INNER JOIN ErrTestTable ON 1 = 1 SET 1 = 1",
```

Comment:
ok, here sqlparser become more accurate. so we need to change this to
```sql
UPDATE TableA INNER JOIN ErrTestTable ON 1 = 1 SET id = 1
```
then, it would be ok.

Reply:
Done in commit 067a5af